stripe/src/test/java/com/stripe/android/view/CardInputWidgetTest.java

```diff
@@ -319,20 +317,16 @@ public void onDeleteFromCvcDate_whenEmptyAndExpiryDateIsEmpty_shiftsFocusOnly()
     }
 
     @Test
-    public void onUpdateIcon_forCommonLengthBrand_callsSetImageResourceAndSetsLengthOnCvc() {
+    public void onUpdateIcon_forCommonLengthBrand_setsLengthOnCvc() {
         // This should set the brand to Visa. Note that more extensive brand checking occurs
         // in CardNumberEditTextTest.
-        Bitmap oldBitmap  = ((BitmapDrawable) mIconView.getDrawable()).getBitmap();
         mCardNumberEditText.append(Card.PREFIXES_VISA[0]);
-        assertNotEquals(oldBitmap, ((BitmapDrawable) mIconView.getDrawable()).getBitmap());
         assertTrue(ViewTestUtils.hasMaxLength(mCvcEditText, 3));
     }
 
     @Test
-    public void onUpdateText_forAmExPrefix_callsSetImageResourceAndSetsLengthOnCvc() {
-        Bitmap oldBitmap  = ((BitmapDrawable) mIconView.getDrawable()).getBitmap();
```

Comment:
No, what changed is that the drawable is no longer a bitmap, so I can't get it out of the ImageView. The drawable never actually changes here, just the resource, but there are no getters for that value. So I can't test it this way (which was always a bit of a hack anyhow)